src/lib/types.ts

```
@@ -401,8 +380,7 @@ export interface ButtonToggleContext {
   isSelected: (toggleValue: string) => boolean;
 }
 
-export interface CheckIconProps extends Omit<SVGAttributes<SVGSVGElement>, "class"> {
-  class?: ClassValue;
+export interface CheckIconProps extends SVGAttributes<SVGSVGElement> {
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)